src/client/common/configSettings.ts

```diff
@@ -705,3 +718,23 @@ function isValidPythonPath(pythonPath: string): boolean {
         return false;
     }
 }
+
+function convertSettingTypeToLogLevel(setting: LoggingLevelSettingType): LogLevel {
```

Comment:
What if it isn't set?

```suggestion
function convertSettingTypeToLogLevel(setting: LoggingLevelSettingType | undefined): LogLevel {
```

Reply:
Then `package.json` returns `"error"` by default.

Reply:
Right, but in some situations (like during tests) defaults from package.json are not applied.

Reply:
Alright

Reply:
Please approve if you find everything else alright.